docs/src/univariate.md

```diff
@@ -481,6 +481,9 @@ DiscreteNonParametric
 Geometric
 Hypergeometric
 NegativeBinomial
+NegativeBinomial2
+NegativeBinomial2Log
+NegativeBinomial3
```

Comment:
I would prefer more descriptive names, similar to `Normal`/`NormalCanon`.

Reply:
I agree, more descriptive names would be nice. A believe a reasonable argument could be made for re-naming `NegativeBinomial3` as `PoissonGamma`. 

For the location-overdispersion parameterization (and the respective log version), it could be re-named uncreatively as `NegativeBinomialAlt` (and `NegativeBinomialAltLog`).

Reply:
Unfortunately, I am not very creative here either and don't have much better suggestions right now :smile: 

Maybe one could use `NegativeBinomialMean` instead of `NegativeBinomial2` to highlight that it uses the mean instead of the number of successes. But I guess the name could maybe be misinterpreted as the mean of a negative binomial distribution.

Reply:
How about
```suggestion
NegativeBinomialLocationScale
NegativeBinomialLogLocationScale
NegativeBinomialShapeScale
```
? that's how you explain them in the PR description, which seems sensible and fairly self-explanatory.

Reply:
An option would be also to add an alias `const PoissonGamma = NegativeBinomialPoissonGamma` for people that prefer the shorter name.

Reply:
With respect to the definition of 'scale parameter', I agree -- ϕ is actually a dispersion parameter in the location parameterizations. Consequently, I will update the wording in the docstrings to reflect this.

Also, I think that the combined proposal of:
```
NegativeBinomialLocation
NegativeBinomialLogLocation
NegativeBinomialPoissonGamma
const PoissonGamma = NegativeBinomialPoissonGamma
```
achieves the best clarity.